pkg/v4/resource/chartconfig/desired.go

```diff
@@ -69,3 +73,35 @@ func newKubeStateMetricsChartConfig(clusterConfig cluster.Config, projectName st
 		},
 	}
 }
+
+func newNodeExporterChartConfig(clusterConfig cluster.Config, projectName string) *v1alpha1.ChartConfig {
+	chartName := "kubernetes-node-exporter-chart"
+	channelName := "0-1-stable"
+	releaseName := "node-exporter"
+
+	return &v1alpha1.ChartConfig{
+		TypeMeta: apismetav1.TypeMeta{
+			Kind:       chartConfigKind,
+			APIVersion: chartConfigAPIVersion,
+		},
+		ObjectMeta: apismetav1.ObjectMeta{
+			Name: chartName,
+			Labels: map[string]string{
```

Comment:
Should there be also `app` and `service-type` labels here?
https://github.com/giantswarm/fmt/blob/master/kubernetes/annotations_and_labels.md#common-labels-for-guest-cluster-services

Reply:
The Chart itself is going to have the labels, e.g. here: https://github.com/giantswarm/kubernetes-node-exporter/blob/cb5cb6e2178216fb47864da29bcc3381f999e5eb/helm/kubernetes-node-exporter-chart/templates/daemonset.yaml#L8

I think we didn't discuss about the CR yet, but would make sense to me too.

Reply:
Good point I'll add. The CR's have the managed-by label because they are created from the host cluster. But having the standard guest labels is a good idea.

Reply:
Done.